packages/autocomplete-js/src/getPanelPositionStyle.ts

```diff
@@ -42,14 +47,14 @@ export function getPanelPositionStyle({
     }
 
     case 'input-wrapper-width': {
-      const inputWrapperRect = inputWrapper.getBoundingClientRect();
+      const formRect = form.getBoundingClientRect();
```

Comment:
is this diff a remainder of #374 or is it calculated on a different element now?

Reply:
That's calculated on the form now, which makes more sense because it's the reference element to base the panel position on.